lang/cs/Org.Apache.REEF.Tang/Implementations/ClassHierarchy/AvroClassHierarchy.cs

```diff
@@ -206,6 +236,33 @@ public INode GetNode(string fullName)
             return ret;
         }
 
+
+        public INode GetNode(string fullName, string aliasLanguage)
```

Comment:
Can you please file the JIRA, mark the relevant parts as `[Osolete]` and with a `//TODO[REEF-xxx]`?